contracts/evm/contracts/xcall/CallService.sol

```diff
@@ -523,4 +525,10 @@ contract CallService is IBSH, ICallService, IFeeManage, Initializable {
     function verifySuccess(uint256 _sn) external view returns (bool) {
         return successfulResponses[_sn];
     }
+    
+        /* ========== UUPS ========== */
+    //solhint-disable-next-line no-empty-blocks
+    function _authorizeUpgrade(address) internal override onlyAdmin {}
+
+    function getImplementation() external view returns (address) {}
```

Comment:
What does this function do?
Should this do something like:
return ERC1967Utils.getImplementation();?